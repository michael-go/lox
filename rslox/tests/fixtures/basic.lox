print 3 * (1 + 2);
print 3 * 1 + 2;
print 5 - 4 > 3 * 2 == !nil;
print !(5 - 4 > 3 * 2 == !nil);
print "abc" == "a" + "bc";
print "abc" == "a" + "bd";
print "foo" + "bar"+"baz";
