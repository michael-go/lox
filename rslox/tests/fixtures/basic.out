# exit code: 0
# stdout:
9
5
false
true
true
false
foobarbaz

# stderr:

